File: 0066-plus-one/0066-plus-one.java
class Solution {
    public int[] plusOne(int[] digits) {

        for (int i = digits.length - 1; i >= 0; i--) {
            if (digits[i] < 9) {
                digits[i]++;
                return digits;
            }
            digits[i] = 0;
        }

        int[] res = new int[digits.length + 1];
        System.out.print(Arrays.toString(res));
        res[0] = 1;
        return res;
    }
}

    

File: 0066-plus-one/0066-plus-one.py
class Solution:
    def plusOne(self, a: List[int]) -> List[int]:
        k=[]
        k1=[]
        v=[]
        s=" "
        for i in range(len(a)):
            a[i]=str(a[i])
            k.append(a[i])
        for i in range(len(k)):
            s=s+k[i]
        s=int(s)+1
        s=list(str(s))
        for i in range(len(s)):
            s[i]=int(s[i])
            v.append(s[i])
        return v
      
        